components/bl-pdfViewer-component/src/index.js

```diff
@@ -5,37 +5,68 @@ import { Controls } from './subcomponents';
 const { cn } = BackendlessUI.CSSUtils;
 
 const sizeVariants = {
-  'small' : { height: 385, width: 250 },
-  'normal': { height: 550, width: 350 },
-  'large' : { height: 630, width: 400 },
+  'small' : { minHeight: 385, width: 250 },
+  'normal': { minHeight: 550, width: 350 },
+  'large' : { minHeight: 630, width: 400 },
 };
 
-export default function PdfViewer({ component }) {
+export default function PdfViewer({ component, eventHandlers }) {
   const { style, display, classList, pdfUrl, renderType, size } = component;
+  const { onLoadSuccess, onLoadError } = eventHandlers;
 
   const [numPages, setNumPages] = useState(null);
-  const [pageNumber, setPageNumber] = useState(1);
+  const [pageIndex, setPageIndex] = useState(1);
```

Comment:
if it is index, so there must be 0
or no?

Reply:
but if the index is 0, there will be a "No page specified" error.